src/nidigital/metadata/enums_addon.py

```diff
@@ -2,5 +2,10 @@
 # Any changes to the API should be made here. enums.py is code generated
 
 enums_override_metadata = {
+    # Codegen deduces the enum types to generate based on usage of enum types in public metadata.
```

Comment:
Do we intend for these things to remain in `_addon.py` files in the long term?

Reply:
No, it will go away when #1066 is fixed.